packages/ndla-ui/src/Messages/MessageBox.tsx

```diff
@@ -63,11 +72,14 @@ const TextWrapper = styled.div`
   }
 `;
 
-const IconWrapper = styled.div`
+const IconWrapper = styled.div<MessageBoxProps>`
   display: flex;
   align-items: flex-start;
-  padding-right: ${spacing.small};
-
+  ${({ noIcon }) =>
+    !noIcon &&
+    css`
+      padding-right: ${spacing.small};
+    `}
```

Comment:
Vi forsøker å unngå å sende props til styled components på grunn av ytelse. Bruk heller data-icon=false.

Nokke sånt blir det då
```suggestion
&[data-icon='false'] {
      padding-right: ${spacing.small};
    }
```

Reply:
ah, good to know! jeg bare fulgte den gamle metoden over. Skal jeg da refaktorere de andre stedene lenger opp der de sender "type" props også?